bugs_and_glitches.md

```diff
@@ -35,8 +35,16 @@ Fixes are written in the `diff` format.
   - [Big Lightning animation has incorrect frame data](#big-lightning-animation-has-incorrect-frame-data)
   - [Dive Bomb animation has incorrect frame data](#dive-bomb-animation-has-incorrect-frame-data)
 - [Text](#text)
-  - [Both Ninetales cards misspell its name](#both-ninetales-cards-misspell-its-name)
+  - [Misspelled "chosen" system text string] (#misspelled-chosen-system-text-string)
+  - [Doctor Mason misspells Exeggcute's name] (#doctor-mason-misspells-exeggcutes-name)
+  - [Clerk's Challenge Cup dialogue has an extra 'the'] (#clerks-challenge-cup-dialogue-has-an-extra-the)
+  - [Ronald's Challenge Cup dialogue has a typo] (#ronalds-challenge-cup-dialogue-has-a-typo)
```

Comment:
All the spaces between the [] and the () break the markdown.

Check here: https://github.com/FrenchOrange/poketcg-irritated/blob/master/bugs_and_glitches.md